app/views/layouts/application.html.erb

```diff
@@ -8,11 +8,9 @@
   </head>
   <body>
     <div class="container">
-      <div>
-        <%= link_to "Add Quote", new_quote_path %>
-        <%= link_to "All Quotes", quotes_path %>
+      <div class="navbar-container">
         <% if current_user %>
-          <%= link_to "Sign Out", destroy_user_session_path, method: :delete %>
+          <%= render "layouts/navbar" %>
```

Comment:
That's another concern Amanda was speaking to.  That maybe we don't want this so that anyone that has a link can just automatically create an account and see everything that's on the site.  Not a huge decision but one we will have to make at some point.